index.bs

```diff
@@ -1954,7 +1955,7 @@ An example of something that was done as a monkey patch
 that is scheduled to be integrated into the web app manifest in a future level (post-CR):
 
 * https://wicg.github.io/web-share-target/#extension-to-the-web-app-manifest
-</div>
+    </div>
```

Comment:
This feels wrong.  Did bikeshed require it?  If so, would adding a blank line between the list item and the `</div>` also fix it?

Reply:
Yeah, bs required it - I'll try adding a blank line instead.